flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/slotpool/PhysicalSlotProviderImplWithSpreadOutStrategyTest.java

```diff
@@ -109,8 +109,10 @@ public void testSlotAllocationFulfilledWithNewSlots()
     @Test
     public void testIndividualBatchSlotRequestTimeoutCheckIsDisabledOnAllocatingNewSlots()
             throws Exception {
-        TestingSlotPoolImpl slotPool =
-                new SlotPoolBuilder(physicalSlotProviderResource.getMainThreadExecutor()).build();
+        DeclarativeSlotPoolBridge slotPool =
+                new DeclarativeSlotPoolBridgeBuilder(
+                                physicalSlotProviderResource.getMainThreadExecutor())
+                        .build();
         assertThat(slotPool.isBatchSlotRequestTimeoutCheckEnabled(), is(true));
```

Comment:
This is an `org.junit.Assert.assertThat` and can be replaced

Reply:
Will fix it.